pkg/skaffold/docker/parse.go

```diff
@@ -123,7 +121,14 @@ func processBaseImage(value *parser.Node) ([]string, error) {
 	return cfg.Config.OnBuild, nil
 }
 
+var imageToConfigCache sync.Map
+
 func retrieveImageConfig(image string) (*manifest.Schema2Image, error) {
+	cachedCfg, present := imageToConfigCache.Load(image)
```

Comment:
What if the image gets overwritten in the registry?

:latest, someone clobbering existing tag, etc.

Reply:
Yeah I thought about that a bit, this kind of matches docker build semantics though. It won't pull an image for updates if it's already pulled locally unless you force it to.

Reply:
Exactly

Reply:
That makes sense, LGTM!